ai-service/tests/conftest.py

```
@@ -16,14 +16,17 @@ def create_db_test_collection(collection_name: str) -> chromadb.Collection:
 
 
 @pytest.fixture(scope="module")
-def db_test_collection(request) -> chromadb.Collection:
-    collection_name = f"{request.module.__name__}"
+def db_test_collection(request: pytest.FixtureRequest) -> chromadb.Collection:
+    module = getattr(request, "module", None)
+    module_name = getattr(module, "__name__")
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)